src/bias/cv.jl

```diff
@@ -0,0 +1,119 @@
+# Calculate collective variables
+
+export
+    CalcDist,
+    CalcRg,
+    CalcRMSD,
+    calculate_cv
+    #calculate_cv_ustrip!, # dont export
+    #cv_gradient #dont export
+
+@doc raw"""
+    CalcDist(atom_idx_1, atom_idx_2, correction)
+
+Bias the distance between two atoms in a system. 
+
+# Arguments
+- `atom_idx_1::Int`: index of the first atom.
+- `atom_idx_2::Int`: index of the second atom.
+- `dist_type::Symbol=:pbc_dist`: distances are evaluated taking periodic boundary conditions into account. Should be `:dist` if PBCs should not be considered.
+- `correction::Symbol=:wrap`: the system is wrapped in the unit cell. Should be `:pbc` to unwrap molecules prior to the distance calculation. 
+"""
+@kwdef struct CalcDist   
+    atom_idx_1::Int     
```

Comment:
Would it be possible to allow multiple indices here and calculate the distance between the centers of mass?